podcasts.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -163,6 +163,15 @@
           "version": "7.25.1"
         }
       },
+      {
```

Comment:
I would really, really avoid adding a new library just for this effect (and at this point).

I already plan to remove the EoY files from the final app build once we remove the feature in January. Of course, we can also remove that but I wonder if it really worth it.